zigzag/zigzag.py

```diff
@@ -44,8 +44,7 @@ def __init__(self, junit_xml_file_path, qtest_api_token, qtest_project_id, qtest
         self._junit_xml_doc = None
 
         self._utility_facade = UtilityFacade(self)
-        parsing_facade = XmlParsingFacade(self)  # no need to attach this to self (mediator)
-        parsing_facade.parse()
+        self._parsing_facade = XmlParsingFacade(self)
```

Comment:
::pestering:: You could save a lot of lines of code if you gated execution of parsing on a boolean flag in the init signature. All the cool kids are doing it!

Reply:
I don't disagree ... I do think that the signature is getting a bit crazy long though.  I would support making that change if/when we moved to a `kwargs` signature.  For right now this is the smallest change that gets us the desired behavior.